openstudiocore/src/openstudio_lib/FacilityStoriesGridView.cpp

```diff
@@ -233,16 +240,16 @@ namespace openstudio {
   {
     //m_itemSelectorButtons->enableAddButton();
     //m_itemSelectorButtons->enableCopyButton();
-    m_itemSelectorButtons->enableRemoveButton();
```

Comment:
@jmarrec this prevents us from deleting stories, is that intended?

Reply:
Copying stories is disabled in last release (assume that is on purpose) but delete is enabled if you select something.  Purge should always be enabled.

Reply:
I may have messed it up. The only thing that should toggle is the `RemoveButton` when selected/unselect, and purge should always be enabled yes (all buttons are enabled by default I believe)